process/process.go

```diff
@@ -60,6 +60,25 @@ type NumCtxSwitchesStat struct {
 	Involuntary int64 `json:"involuntary"`
 }
 
```

Comment:
Could you add a comment about where these constants comes from? for example  `sys/resource.h in Linux`  or appropriate url.